src/odemis/driver/xt_client.py

```diff
@@ -2097,22 +2106,11 @@ def _moveTo(self, future, pos, timeout=60):
                 self._updatePosition()
 
     def _doMoveRel(self, future, shift):
-        pos = self._getPosition()
-        for k, v in shift.items():
-            pos[k] += v
-
-        target_pos = self._applyInversion(pos)
-        # Check range (for the axes we are moving)
-        for an in shift.keys():
-            rng = self.axes[an].range
-            if rng == (0, 2 * math.pi) and an in ("rx", "rz"):
-                pos[an] = pos[an] % (2 * math.pi)
-                target_pos[an] = target_pos[an] % (2 * math.pi)
-            p = target_pos[an]
-            if not rng[0] <= p <= rng[1]:
-                raise ValueError("Relative move would cause axis %s out of bound (%g m)" % (an, p))
-
-        self._moveTo(future, pos)
+        """
+        shift (dict): position in internal coordinates
```

Comment:
what are internal coordinates?

Reply:
The user can invert the direction of an axis (with the parameter `inverted`). That's what `applyInversion()` takes care of. "Internal coordinates" mean *without* the inversion applied, ie the hardware point-of-view.